src/page-components/collections/collection-chapter.astro

```diff
@@ -0,0 +1,26 @@
+---
+import styles from "./collection-chapter.module.scss";
+
+interface CollectionChapter {
+    num: number;
+    title: string;
+    description: string;
+    href: string;
+}
+
+const {num, title, description, href} = Astro.props as CollectionChapter;
+---
+
+<div class={styles.container}  onclick={href && `location.href='${href}'`}>
```

Comment:
I've noticed that parts of this div (the chapter description, below the title/`<a>` heading) are clickable via the `onclick=` but don't display the cursor pointer icon. Not sure what the best practice is for that, but it seemed confusing to me so I thought I'd mention.

Reply:
> While links may have other visual clues to indicate that they are links (such as underlines and special placement), displaying the hand pointer on hover is the definitive indication of a link. To avoid confusion, it is imperative not to use the hand pointer for other purposes

https://ux.stackexchange.com/a/105027

This is an endless debate amongst many, but I think we're in the clear here haha